packages/braintree-integration/src/braintree-fastlane/braintree-fastlane-payment-strategy.ts

```diff
@@ -90,6 +101,7 @@ export default class BraintreeFastlanePaymentStrategy implements PaymentStrategy
         this.initializeCardComponent();
 
         braintreefastlane.onInit((container) => this.renderBraintreeAXOComponent(container));
+        braintreefastlane.onChange && braintreefastlane?.onChange(() => this.handleBraintreeStoredInstrumentChange(methodId));
```

Comment:
Let's go other way. Because if you pass onChange callback from checkout ui for Connect feature, then you might get an error while loading Braintree Fastlane

```suggestion
    if (this.isFastlaneEnabled && braintreefastlane.onChange && typeof braintreefastlane.onChange === 'function') {
                braintreefastlane.onChange(() => this.handleBraintreeStoredInstrumentChange(methodId));
    }
```

Reply:
Yes, you are right! Added condition